app/src/androidTestE2eTest/java/io/appium/uiautomator2/unittest/test/DeviceCommandsTest.java

```diff
@@ -69,12 +69,11 @@
 
 @SuppressWarnings("JavaDoc")
 public class DeviceCommandsTest extends BaseTest {
-
     /**
      * Test for findElement
      */
     @Test
-    public void findElementTest() {
+    public void zzzFindElementTest() {
```

Comment:
Could you leave a comment on why `zzz` is here?
https://github.com/appium/appium-uiautomator2-server/pull/340#issuecomment-616229653

Reply:
Absolutely. Hopping on a call in a few minutes, but I have some free time immediately after that which should be enough to get both the comment here and the markdown updated.

Thanks for your review(s)

Reply:
probably a comment and a `// TODO figure out test state ordering issue`

Reply:
This will be resolved w/ the upcoming update(s)